python/ray/autoscaler/updater.py

```diff
@@ -215,13 +217,15 @@ def do_update(self):
         self.provider.set_node_tags(self.node_id,
                                     {TAG_RAY_NODE_STATUS: "setting-up"})
 
+        m = "{}: Startup commands completed".format(self.node_id)
```

Comment:
```suggestion
        m = "{}: Initialization commands completed".format(self.node_id)
```

just in case bc strings are easy to miss